daisy_workflows/image_build/debian/debian_fai.wf.json

```diff
@@ -5,21 +5,15 @@
     "build_date": {"Value": "${DATE}", "Description": "Build datestamp used to version the image."},
     "debian_cloud_images_version": {"Required": true, "Description": "The debian-cloud-images scripts git commit ID or branch to use."},
     "debian_version": {"Required": true, "Description": "The FAI tool debian version to be requested."},
-    "builder_source_image": {"Required": true, "Description": "The image used to run fai-tool."},
+    "builder_source_image": {
+      "Value": "projects/compute-image-tools/global/images/family/debian-10-worker",
```

Comment:
it's fine for this to be parameterized, but code the default into the base rather than repeating for each invocation